app/src/main/java/chat/rocket/android/push/PushManager.kt

```diff
@@ -383,12 +395,12 @@ data class PushMessage(
 ) : Parcelable {
 
     constructor(parcel: Parcel) : this(
+        parcel.readString()!!,
```

Comment:
🛑 Not so good using `!!`

Reply:
The nullable `parcel.readString()` was ignored before. Now I handled with elvis operator which returns empty if null.